### ISSUE TITLE
Change or customize default filters view

### ISSUE THREAD

**Issue by stevejburge** [upstreamplugin/UpStream#544]
> I would like to set the default view to NOT show completed or closed tasks. At the moment, the only way I seem to be able to do this is EVERY time I go to the project page, select ALL OTHER statuses.

This makes sense to me. By default, closed or completed tasks should be hidden.

**Comment by b-trav** [comment]
Hi @stevejburge and @andergmartins, just to give another use-case: When I go to a project in the front-end, I need to press "Toggle Filters" then add "None" and "In Progress" to the status, just to see the current list of open tasks. If I then add a task, I get the whole list again, so have to apply the filters.

Do you use this plugin as a project management system? If so, what do you do with the large list of completed tasks?

Any hints on fixing this would be greatly appreciated, as the plugin is a little unworkable without this fix. I am happy to delve into the plugin code if you can point me in the right direction.

**Comment by b-trav** [comment]
I believe that a solution of this for projects as well as tasks would solve issue #236.

It would be nice to be able to mark a project as "closed" and then not see it on the front-end list by default.

**Comment by stevejburge** [comment]
@andergmartins One more customer request for this:

> I'm curious. Is there a way to create a default filter? For example, when you first load a project page, there are no filters. I'm curious to know if I could potentially have the page load and only show "in progress" tasks without having to toggle filters.

**Comment by b-trav** [comment]
Hi, this issue was labeled as "high priority" back in July. I am wondering if there has been any progress?

The only way that my group can use your plugin at the moment is by my manually inserting the following code into tasks.php after EVERY upgrade.
```
//Only show open tasks
    $new_data = [];
    foreach ( $rowset as $key => $value ) {
        if ( $statuses[$value["status"]]["type"] != "closed" ) $new_data[$key] = $value; 
    }
    $rowset = $new_data;
```

Otherwise the plugin is quite unusable, as every time you add a task, or make any change, you are overwhelmed by 'closed' tasks, as all the filters reset.

**Comment by stevejburge** [comment]
Fixed in 1.22 https://upstreamplugin.com/upstream-1-22/
